### PR TITLE
switch from brunch to webpack closes #13

### LINKED ISSUE #13: Replace brunch with webpack
Phoenix will use webpack in the next release https://github.com/phoenixframework/phoenix/pull/2779

### PR DESCRIPTION
**Pull request by willfore**
Updated package to use npm's webpack build function for building assets. Updated to the 0.7 version of Bootleg. Updated the readme to reflect the changes needed.

### CODE REVIEW

**Comment by brienw** [comment]
Thanks for the PR, @willfore! Can you put the Elixir version down to 1.6 to match `bootleg`'s current minimum (we've kept it there to match `distillery`'s min) and there's some tricky version matching required between OTP and Elixir right now, so you'll have to set `prepare.sh` to use:
```
export ERLANG_VERSION="20.3"
export ELIXIR_VERSION="v1.6.0"
```
i suspect that will fix the reason the build is failing

Thanks again for making this change!

**Comment by willfore** [comment]
No problem. All done! :)

File: mix.exs [hunk 1]
```diff
@@ -5,19 +5,20 @@ defmodule BootlegPhoenix.Mixfile do
...
-     docs: [source_ref: "v#{@version}", main: "readme", extras: ["README.md"]],
-     test_coverage: [tool: ExCoveralls],
-     dialyzer: [plt_add_deps: :transitive, plt_add_apps: [:mix, :sshkit]],
-     package: package(),
-     description: description(),
-   ]
+    [
+      app: :bootleg_phoenix,
+      version: @version,
+      elixir: "~> 1.7",
```
Comment (brienw) [inline mix.exs]: Oops, please bump this down to 1.6 as well!

**Comment by brienw** [comment]
FYI: the build appears to be failing because the build cache is not being properly rebuilt, i'm looking into why that is and will hopefully have it sorted soon

**Comment by brienw** [comment]
This new CI failure is due to more recent versions of `credo` getting really tripped up over some of our internal meta programming in `bootleg`. once sec, i'll adjust the credo config for `bootleg_phoenix`

**Comment by brienw** [comment]
@willfore I'm unable to push to your branch (this could be a config error on my side, i'm on a new laptop that i haven't fully set up yet), so could you just change line 86 of `.credo.exs` from:
`        {Credo.Check.Readability.StringSigils},`
to:
`        {Credo.Check.Readability.StringSigils, false},`

**Comment by willfore** [comment]
Sure thing, incoming.

**Comment by brienw** [comment]
@willfore I've adjusted some package dependencies versions and adjusted some of the testing infrastructure to support this. Can you please add the necessary files to the fixture `test/fixtures/drunkin_phoenix_1_4` so that it has something to build. After that is set, I think the only remaining thing is to add a note to the README.md about using older version of `bootleg_phoenix` for use with Phoenix < 1.4

You can run the tests locally via `MIX_ENV=test mix test` and you will need to make sure you have Docker installed. Lemme know if you have questions or need any help!

**Comment by willfore** [comment]
I added all the needed files to match the changes. I did not update the README in regards to using with Phoenix < 1.4 as I wasn't sure about version numbers. Let me know if anything else needs to be done.

**Comment by brienw** [comment]
This is great, thank you! I'll update the README today

**Pull request closed without merging**